src/cpu/core_dyn_x86/cache.h

```diff
@@ -211,7 +215,9 @@ class CodePageHandler : public PageHandler {
 		}
 		addr&=4095;
 		if (host_readw(hostmem+addr)==(Bit16u)val) return false;
-		if (!*(Bit16u*)&write_map[addr]) {
```

Comment:
PPC: Fix needs to be aligned and native endian (not swapped to little-endian like the `host_*` functions do).